examples/SLED/entities/Ilness.entity.json

```diff
@@ -0,0 +1,33 @@
+{
+  "info": {
+    "title": "Ilness",
```

Comment:
@prolonMarkLogic  Please correct spelling for file name, definition and title

Reply:
@kkanthet this change was fixed